Cargo.toml

```diff
@@ -22,4 +22,11 @@ tokio-diesel = "0.3.0"
 tracing = "0.1.13"
 tracing-subscriber = "0.2.4"
 
-near-indexer = { git = "https://github.com/nearprotocol/nearcore", rev="830c2149fc02ee23e6e26d198ac12b16c1858eeb" }
\ No newline at end of file
+near-indexer = { git = "https://github.com/nearprotocol/nearcore", rev="c9c6cc3f3c34781fe118a0b76d1cfb0a36f15545" }
+near-store = { git = "https://github.com/nearprotocol/nearcore", rev="c9c6cc3f3c34781fe118a0b76d1cfb0a36f15545" }
```

Comment:
If we merge this, let's have a comment on these dependencies that this is a shortcut we take.